vagrant.rb

```diff
@@ -50,15 +50,16 @@ def configure(config, os, pf_name, pf_id, host_name, host_id, setup, version, se
   net = "192.168." + (pf_id+40).to_s
   ip = net + "." + (host_id+2).to_s
   forward = 100*(80+pf_id)+80
-  command  = '/vagrant/scripts/cleanbox ' + net + ' "' + host_list + '"\n'
+  command_array  = [ %Q(/vagrant/scripts/cleanbox #{net} "#{host_list}") ]
   if setup == "server" then
-    command += 'export ALLOWEDNETWORK=' + net + '.0/24\n'
-    command += '/vagrant/scripts/rudder-setup setup-server "' + version + '"\n'
-    command += '/vagrant/scripts/create-token\n'
+    command_array.push( %Q(export ALLOWEDNETWORK=#{net}.0/24
+/vagrant/scripts/rudder-setup setup-server "#{version}"
+/vagrant/scripts/create-token))
```

Comment:
Either you use newlines in a normal string or you use an array of single lines, but you should not use both